README.adoc

```diff
@@ -96,6 +96,12 @@ image:docs/images/file_builder.png[builder]
 ** Make sure about `import` libraries used in the code installed properly
 ** If there is any HTML/Images as outputs, all will be saved in the workspace under the Machine Leaning task folder
 
+=== Troubleshooting
+
+If you are having trouble viewing the HTML files saved, check your browser console to see if there are any errors about Content Security Policy.
+This is often a culprit.
```

Comment:
This is often a culprit -> This is a known issue ? Or this is often the problem.